.travis.yml

```diff
@@ -29,6 +29,7 @@ env:
   - LAYER=mailparse PHP="72 73 74"
   - LAYER=mongodb PHP="72 73 74"
   - LAYER=msgpack PHP="72 73 74"
+  - LAYER=newrelic PHP="72 73 74"
```

Comment:
This is not needed any more. But can you update `.github/workflows/ci.yml`?

Reply:
Will do. This was in the documented steps which is why I did it.

Reply:
It was true at the time you made the PR. But I've been doing a lot of changes last 10 days.

Reply:
:) Not a problem happens from time to time and great to hear you've been adding stuff 👍